discovery-ui/values.yaml

```diff
@@ -1,21 +1,18 @@
+global:
+  auth:
+    auth0_domain: ""
+  ingress:
+    rootDnsZone: localhost
+    dnsZone: localhost
+    port: 80
```

Comment:
Probably need to remove the port from here.  I couldn't find any references to `global.ingress.port` in this PR

Reply:
Removed